lib/scripts.js

```diff
@@ -65,7 +65,7 @@ var scripts = {
       return result === 1;
     });
   },
-  addJob: function(client, toKey, lifo, job){
+  addJob: function(client, toKey, lifo, customJobId, job){
     var jobArgs = _.flatten(_.toPairs(job));
```

Comment:
since lifo is optional and customId also, I think it is more convenient to put it in an opts argument, and also move it to the last argument in the list:```js function(client, toKey, job, opts) ```